src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String PATH_DATA_FILE = "src/test/java/resources/dataTest.csv";
+    private static final String PATH_EMPTY_FILE = "src/test/java/resources/emptyTest.csv";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void readFile_ok() {
```

Comment:
2. Використовуйте однаковий шаблон найменування для всіх тестових методів в усіх тестових класах (окрім методів @Before, @BeforeClass, @After, @AfterClass). Він може складатися з двох або трьох частин. Рекомендую використовувати наступний шаблон:
<methodUnderTest>_<state>_<expectedBehavior>
Maybe it will be like "readFile_testingFile_ok"?